test/SmokeTests/SmokeTestsV1/VSSample.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
+    <AzureFunctionsVersion>v1</AzureFunctionsVersion>
+  </PropertyGroup>
+  <!-- Common packages for all targets -->
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.4.1" />
```

Comment:
To make this process even smoother, add a project reference directly to the extension project. That way you don't need to deal with nuget packages at all. It makes it waaay easier to just quickly F5 and go.